Source/gendung.cpp

```diff
@@ -549,4 +549,52 @@ void InitLevels()
 	setlevel = false;
 }
 
+void FindTransparencyValues(uint8_t tileID, int i, int j, int x, int y, int d)

```

Comment:
Probably it would have better to add the new parameter to the end instead of the start, also tileID isn't very descriptive, I'm guessing that this is the ID of the floor tile for the given tilset so a better name might have been floorID

Reply:
opened a new pull request to address this issue #2555. Thank you!